### DISCUSSION TITLE
json-schema is vulnerable to Prototype Pollution

### DISCUSSION THREAD

**Discussion by NicestRudeGuy** [json-schema-org/community#376] [General]
So i recently migrated a old CRA design system with storybook to vite to fix vulnerabilities 
any idea when this will be fixed or some alternatives ?

<img width="730" alt="image" src="https://user-images.githubusercontent.com/9863419/231969760-80d753b1-3b1e-4b37-b7af-217a74341828.png">

**Comment by gregsdennis** [comment]
Hey, thank for the info.  I think you're talking about an implementation called `json-schema`.  This repo is for the specification itself. We don't manage any implementations here.

You'll want to report this on the repo for that implementation.

**Comment by jdesrosiers** [comment]
The appropriate repo to report this problem is https://github.com/kriszyp/json-schema.
